deployment/artifacts/mainnet.json

```diff
@@ -3666,9 +3666,9 @@
                     "outputs": []
                 }
             ],
-            "tx_hash": "0x7858c6f34297752433f772ef6b7fa7086608ac3207fc9df5d4d6136ed7ab9b1f",
-            "block_number": 70019855,
-            "deployer": "0x1591165F1BF8B73de7053A6BE6f239BC15076879"
+            "tx_hash": "0xbfdb1ba47830e495dcae80c93943c4a0be539f0e94f795f0330d8e33e6c025fb",
+            "block_number": 70938418,
+            "deployer": "0x0224B7B41E9204550b8B3Fdc1afd6200446576E8"
```

Comment:
This part shouldn't have changed for a proxy upgrade, no? Or am I mistaken (cc @KPrasch )

Reply:
IIRC, our current default is to override these values with the latest implementation data, but it's true that it's not technically necessary.